easybuild/easyconfigs/f/FreeFem++/FreeFem++-3.58-foss-2017b-downloaded-deps.eb

```diff
@@ -1,4 +1,4 @@
-easyblock = 'ConfigureMake'
+easyblock = 'EB_FreeFem_plus__plus_'
```

Comment:
@hajgato This should not be needed, will be auto-derived based on software name...

Reply:
Done